JSQMessagesViewController/Views/JSQMessagesInputToolbar.h

```diff
@@ -83,6 +83,17 @@
 @property (assign, nonatomic) BOOL sendButtonOnRight;
 
 /**
+ *  Specify if the send button might be enabled.
+ *
+ *  @discussion Normally the send button will be enabled if the `textView` contains text. This
+ *  property overrides this behaviour, for example to disable the send button, if the internet
+ *  connection is lost.
+ *
+ *  @see `updateSendButtonEnabledState`.
+ */
+@property (assign, nonatomic) BOOL sendButtonEnabled;
```

Comment:
I'm hesitant because setting this property also immediately disables/enables the send button (given there's text in the text view). If the property is called `automaticallyUpdatesSendButtonState` I'd expect the implementation of `updateSendButtonEnabledState` to return early, if this property is set to `NO` instead of it being `AND`ed with `hasText`. But then a user would have to set `automaticallyUpdatesSendButtonState = NO` _and_ manually disable the send button which is two instructions instead of one for the (probably) most common use case "I want to disable the send button, no matter what". 

So maybe `sendButtonDisabled`? The negation here would solve the "setting it to `YES` does not necessarily enable the button"-problem.